src/main/java/org/folio/support/ExpirationTool.java

```diff
@@ -1,13 +1,5 @@
 package org.folio.support;
 
-import java.text.SimpleDateFormat;
```

Comment:
Please can these changes be reset to use the ordering defined in https://issues.folio.org/browse/FOLIO-1667